package.json

```diff
@@ -20,8 +20,8 @@
     "format:check-fix:prettier": "run-e format:check:prettier format:fix:prettier",
     "format:check:prettier": "cross-env-shell prettier --check $npm_package_config_prettier",
     "format:fix:prettier": "cross-env-shell prettier --write $npm_package_config_prettier",
-    "test:dev": "ava",
-    "test:ci": "nyc -r lcovonly -r text -r json ava"
+    "test:dev": "ava --serial",
```

Comment:
This is temporary while we have two main test files (`main.js` and `main_esbuild.js`). Some of the tests do things like renaming and deleting files, which becomes problematic when the tests run concurrently.

Reply:
Should we use `test.serial()` on each of those tests instead?
This would ensure running as many tests concurrently as possible, to speed them up.

Reply:
I tried that before and it didn't work. I'll give it another try.

Reply:
I solved it in https://github.com/netlify/zip-it-and-ship-it/pull/325/commits/e58cb990a00b24adcddbc2f4227670bb9c985744 by using different temporary directory prefixes for the two bundles, and using different fixtures for the problematic tests so that we don't have race conditions on the same files.